crates/fsrt/src/main.rs

```diff
@@ -99,7 +103,69 @@ impl fmt::Display for Error {
     }
 }
 
-impl std::error::Error for Error {}
+fn check_graphql_and_perms(val: &Value) -> Vec<&str> {
```

Comment:
What's the string being returned here? And does it need to be a string in the first place?

Reply:
This goes back to my internship, we decided on using string instead of enums for returning permissions.... I suppose we can swtich that. Let me know if we should.

Reply:
The main reason I'm going on this now, is because you have to hardcode the permissions anyway with the mapping, which means that the permissions are known at compile time.

Though if that's a stand in for deserializing an actual permission map where the permissions aren't static, then you can leave it as str, though make sure to add a comment.

Reply:
Yeah, i haven't completely decided on how I'll fetch them. I'll either be parsing the nadel files or just creating a map.... we can discuss this.

Reply:
Eventually, I want this to be enums, but I think we should make that change in a different task. Let me know if that sounds good :)